CollectPoints/Logic/vtkSlicerCollectPointsLogic.cxx

```diff
@@ -257,7 +303,7 @@ void vtkSlicerCollectPointsLogic::ProcessMRMLNodesEvents( vtkObject* caller, uns
   {
     if ( collectPointsNode->GetCollectMode() == vtkMRMLCollectPointsNode::Automatic )
     {
-      if ( collectPointsNode->GetOutputNode() == NULL || collectPointsNode->GetProbeTransformNode() == NULL )
+      if ( collectPointsNode->GetOutputNode() == NULL || collectPointsNode->GetSamplingTransformNode() == NULL )
       {
         vtkWarningMacro( "Collect fiducials node is not fully set up. Setting to manual collection." );
         collectPointsNode->SetCollectModeToManual();
```

Comment:
I *think* the reason I did this was because the anchor transform is being observed. If the sampling transform is null but the anchor transform is changing rapidly (through OpenIGTLink or another module), *and* we're in auto-collect mode, then this function would be run several times a second. Meaning a warning message is logged several times every second. In hindsight, this might not be a major issue. Should I change this so that it doesn't change the collect mode?

Reply:
https://github.com/SlicerIGT/SlicerIGT/pull/170/files/ea86a7b8f892cb3befedd5161a26c8a3084cf0c4..1b9a0e0a4521ba12bcbb40916c16420a7436bcf9#r174669150

I wonder now if the anchor transform even needs to be observed. The idea is that the anchor transform provides a basis for points. Should we collect points whenever the anchor moves?